src/main/kotlin/com/healthmetrix/labres/lab/UpdateResultUseCase.kt

```diff
@@ -26,9 +27,9 @@ class UpdateResultUseCase(
             )
         )
 
-        if (saved.notificationId != null)
+        if (saved.notificationId != null && saved.status != Status.IN_PROGRESS)
             notifier(saved.notificationId)
-        else
+        else if (saved.notificationId == null)
```

Comment:
this will def conflict with my PR: https://github.com/labres/server/pull/35/files#diff-7a83d531d82532d86ec6c962d62e3f7aR21-R27

not a big deal but just something to keep in mind